templates/adding_a_new_example_script/utils_xxx.py

```diff
@@ -868,8 +868,6 @@ def write_predictions_extended(
         orig_data = json.load(reader)["data"]
 
     qid_to_has_ans = make_qid_to_has_ans(orig_data)
-    has_ans_qids = [k for k, v in qid_to_has_ans.items() if v]
-    no_ans_qids = [k for k, v in qid_to_has_ans.items() if not v]
```

Comment:
I'm not seeing any example scripts containing similar code so it doesn't look useful to keep in the template.